components/Button.js

```diff
@@ -26,6 +53,10 @@ Button.propTypes = {
   ...TouchableOpacity.propTypes,
 };
 
+Button.defaultProps = {
+  inProgress: undefined,
```

Comment:
`false` rather than undefined, just to make it a little bit cleaner.

Also spread `TouchableOpacity.defaultProps` in `Button.defaultProps`.